assets/src/dashboard/app/index.js

```diff
@@ -47,23 +48,28 @@ function App({ config }) {
             <RouterProvider>
               <GlobalStyle />
               <KeyboardOnlyOutline />
-              <Route
-                exact
-                path={APP_ROUTES.MY_STORIES}
-                component={<MyStoriesView />}
-              />
-              <Route
-                path={APP_ROUTES.TEMPLATE_DETAIL}
-                component={<TemplateDetail />}
-              />
-              <Route
-                path={APP_ROUTES.TEMPLATES_GALLERY}
-                component={<TemplatesGalleryView />}
-              />
-              <Route
-                path={APP_ROUTES.MY_BOOKMARKS}
-                component={<MyBookmarksView />}
-              />
+              <AppFrame>
+                <LeftRail />
+                <PageContent>
+                  <Route
+                    exact
+                    path={APP_ROUTES.MY_STORIES}
+                    component={<MyStoriesView />}
+                  />
+                  <Route
+                    path={APP_ROUTES.TEMPLATE_DETAIL}
+                    component={<TemplateDetail />}
+                  />
+                  <Route
+                    path={APP_ROUTES.TEMPLATES_GALLERY}
+                    component={<TemplatesGalleryView />}
+                  />
+                  <Route
+                    path={APP_ROUTES.MY_BOOKMARKS}
+                    component={<MyBookmarksView />}
+                  />
+                </PageContent>
```

Comment:
Nit, did we want to change the labels to the new designs in this PR?

"My Stories" -> stays the same
"Bookmarks" -> "Saved Templates"
"Templates Gallery" -> "Explore Templates"

(Taken from a screenshot of the design movies)
![image](https://user-images.githubusercontent.com/40646372/80012784-b2e21f80-8482-11ea-8b2b-899cc5b22d6c.png)